Library/Homebrew/github_packages.rb

```diff
@@ -115,33 +120,42 @@ def upload_bottle(user, token, skopeo, formula_name, bottle_hash)
       ".#{rebuild}"
     end
     version_rebuild = "#{version}#{rebuild}"
-    root = Pathname("#{formula_name}-#{version_rebuild}")
+    root = Pathname("#{formula_name}--#{version_rebuild}")
     FileUtils.rm_rf root
 
     write_image_layout(root)
 
     blobs = root/"blobs/sha256"
     blobs.mkpath
 
-    formula_path = HOMEBREW_REPOSITORY/bottle_hash["formula"]["path"]
-    formula = Formulary.factory(formula_path)
-
     # TODO: ideally most/all of these attributes would be stored in the
     # bottle JSON rather than reading them from the formula.
     git_revision = formula.tap.git_head
     git_path = formula_path.to_s.delete_prefix("#{formula.tap.path}/")
     source = "https://github.com/#{org}/#{repo}/blob/#{git_revision}/#{git_path}"
+    documentation = if formula.tap.core_tap?
+      "https://formulae.brew.sh/formula/#{formula_name}"
+    else
+      formula.tap.remote
+    end
 
     formula_annotations_hash = {
-      "org.opencontainers.image.description" => formula.desc,
-      "org.opencontainers.image.license"     => formula.license,
-      "org.opencontainers.image.revision"    => git_revision,
-      "org.opencontainers.image.source"      => source,
-      "org.opencontainers.image.url"         => formula.homepage,
-      "org.opencontainers.image.vendor"      => org,
-      "org.opencontainers.image.version"     => version,
+      "org.opencontainers.image.description"   => formula.desc,
+      "org.opencontainers.image.documentation" => documentation,
+      "org.opencontainers.image.license"       => formula.license,
+      "org.opencontainers.image.ref.name"      => version_rebuild,
+      "org.opencontainers.image.revision"      => git_revision,
+      "org.opencontainers.image.source"        => source,
+      "org.opencontainers.image.title"         => formula.full_name,
```

Comment:
What's this used for?

Reply:
It's the human readable name of the image. I'm hoping that the GitHub web interface would display it.